pkg/crypto/crypto.go

```diff
@@ -61,17 +64,33 @@ func encrypt(plaintext string, key []byte) (string, error) {
 	writer.Close()
 	encoder.Close()
 
-	return string(msgBuffer.Bytes()), nil
+	// Remove unnecessary characters and trim new lines before storing as
+	// part of node Annotations username.
+	encryptedStr := string(msgBuffer.Bytes())
+	encryptedStr = strings.Replace(encryptedStr, startTag, "", -1)
+	encryptedStr = strings.Replace(encryptedStr, endTag, "", -1)
+	encryptedStr = strings.Trim(encryptedStr, "\n")
+	return encryptedStr, nil
 }
 
 // decrypt converts encrypted contents to plaintext using the given key as a passphrase
 func decrypt(encrypted string, key []byte) (string, error) {
 	if key == nil {
 		return "", errors.New("decryption passphrase cannot be empty")
 	}
+	// As we trim and remove unnecessary characters during encrypt call, we need to
```

Comment:
How is this going to handle existing nodes from the previous WMCO version, which have the previous formatting, the start/end tags in the annotation?

Reply:
In addition we could also check the annotation in e2e test that it does not have the tag.

Reply:
I am not able to follow up the ask here. we already have function to test encrypt/decrypt .  Should I add similar one to test DecryptFromJSONString /EncryptToJSONString calls ?

Reply:
added new UT to cover this case.